src/js/components/TabForm.js

```diff
@@ -173,9 +173,9 @@ class TabForm extends mixin(InternalStorageMixin) {
       let formRowClassSet = classNames('row', formRowClass, formKey);
 
       return (
-        <div key={i} className="form-panel">
+        <div key={i} className={formPanelClassSet}>
           <Form
-            className={panelClassSet}
+            className="form"
```

Comment:
Had a conversation offline with @jfurrow. It seem that this is the only place we are using a `form` class and it is only being defined in `multiple-form.less`, so @jfurrow is now exploring a solution where we use `form`-tag instead